src/component/feature/feature-list-item-component.jsx

```diff
@@ -42,27 +43,61 @@ const Feature = ({
             <span className={styles.listItemMetric}>
                 <Progress strokeWidth={15} percentage={percent} isFallback={isStale} />
             </span>
-            <span className={styles.listItemToggle}>
-                <Switch
-                    title={`Toggle ${name}`}
-                    key="left-actions"
-                    onChange={() => toggleFeature(name)}
-                    checked={enabled}
-                />
-            </span>
+            {toggleFeature ? ( // display feature list, toggleFeature is available
```

Comment:
We end up with a lot of ternary operators here to see if it is a list of archived toggles or active toggles. Would it be better to split it in two components, one for archived and one for active, and split out shared parts out as smaller components?